recipe/meta.yaml

```diff
@@ -15,15 +15,14 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - wheel
-    - setuptools >=42
+    - setuptools >=56
     - setuptools_scm >=3.4.1
     - toml
   run:
     - python >=3.6
-    - importlib_metadata >=0.5  # [py<38]
```

Comment:
I'm not 100% convinced that this should be removed. `importlib.metadata` was added to the standard library in Python 3.8, and this shim package may still be required to make this noarch package work for Python <3.8 environments.

Reply:
You'll probably have to dig into the upstream code to see if/when importib.metadata is (unconditionally) imported. A test should be added to the recipe to make sure this package doesn't fail for Python <3.8.

Reply:
The usage of importlib_metadata was removed completely in v15.0.0 https://github.com/jaraco/path/commit/cfc337705e54e68d17ba498f0b2b69110692a8d4#diff-80d7b10846b7c6414cdd9a56616abb53bc91dce8f615f6fd6c71763d7f19cc16

No mentions about importlib_metadata in v16.2.0.
`rg "importlib"`
```
CHANGES.rst
44:  use ``importlib.metadata.version('path')``.
122:- #174: Rely on ``importlib.metadata`` on Python 3.8.
196:- #146: Rely on `importlib_metadata
197:  <https://pypi.org/project/importlib_metadata>`_ instead of
201:  importlib_metadata package, but the project still supports

test_path.py
26:import importlib
1238:        appdirs = importlib.import_module('appdirs')
1294:        appdirs = importlib.import_module('appdirs')

path/__init__.py
38:import importlib
1497:        appdirs = importlib.import_module('appdirs')
```
And changelog https://github.com/jaraco/path/blob/v16.2.0/CHANGES.rst